tools/noise/subr.sh

```diff
@@ -450,7 +450,7 @@ export_large_jpeg() {
 	tool_installed darktable-cli
 
 	rm -f "$output" "$xmp"
-	darktable-cli "$input" "$output" &> /tmp/dt_output.log  || echo $(cat /tmp/dt_output.log); exit 1
+	darktable-cli "$input" "$output" &> /tmp/dt_output.log  || (echo $(cat /tmp/dt_output.log); exit 1)
```

Comment:
BTW, isn't this equivalent:
` ...  || ( cat /tmp/dt_output.log); exit 1 )`

I really don't see the point of the echo except if there is some variables references `$VAR` in the log and we want to eval them given current environment, but I doubt it is the case here.

Reply:
Probably (maybe there were some `$VAR` in the output log back in darktable 1.1.

Reply:
So let's change that. TIA.

Reply:
I did that not too long ago, the idea was to keep darktable quiet unless it returns with an error code signaling that the export was not successful. 
In my case back then I run the script while I still had darktable open and the script silently stopped working without giving me this simple warning:
```
[init] the database lock file contains a pid that seems to be alive in your system: 2530733
[init] database is locked, probably another process is already using it
ERROR: can't acquire database lock, aborting.
```
And then all the following steps stop making sense, hence the `exit 1`. But I agree placing the `exit 1` in a sub-function is not nice, this should be done in the main loop within `darktable-gen-noiseprofile`. I'll try to come up with a solution.

In the meantime I found the following info: darktable-cli uses `--library :memory:` by default, so why should darktable-cli check whether the system user library is locked by another darktable process anyway?
@TurboGit do you know something about it?